### PR TITLE
Iconography: Security

### PR DESCRIPTION
**Pull request by Michele-Camargo**
# What

Implement the new icons proposed in the [Iconography](https://www.figma.com/file/9WVVk8Vobz2ZMzmHjtEFEh/Iconography?node-id=1825%3A1490) spec

# Why

Enable us to have a greater visual collection, once we have a greater and more diverse amount of icons

# How

Added informations components in /components/Icons

# Sample

https://user-images.githubusercontent.com/44146877/169369236-68770401-62f8-470e-9430-74bd9db0a701.mov

# QA

### CODE REVIEW

File: src/components/Icons/Security/PadlockLock.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+
+import BaseIcon from '../BaseIcon';
+import { getFill, getSize, getViewBox } from '../utils';
+import type { IconProps } from '../types';
+
+function PadlockLock({
```
Comment (lucianomlima) [inline src/components/Icons/Security/PadlockLock.tsx:9]: ⛏️ Acho que fica mais condizente e de melhor leitura se mudarmos para `PadlockOpen` e `PadlockClosed`

File: src/components/Icons/Security/ShieldError.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+
+import BaseIcon from '../BaseIcon';
+import { getFill, getSize, getViewBox } from '../utils';
+import type { IconProps } from '../types';
+
+function ShieldCheck({
+  id = 'ShieldCheckIcon',
```
Comment (lucianomlima) [inline src/components/Icons/Security/ShieldError.tsx:10]: 🔧 Parece que o nome desse arquivo está errado

File: src/components/Icons/Security/ShienldError.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+
+import BaseIcon from '../BaseIcon';
+import { getFill, getSize, getViewBox } from '../utils';
+import type { IconProps } from '../types';
+
+function ShieldError({
+  id = 'ShieldErrorIcon',
```
Comment (lucianomlima) [inline src/components/Icons/Security/ShienldError.tsx:10]: 🔧 O nome desse arquivo também está incorreto

File: src/components/Icons/Security/index.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+export { default as FingerprintErrorIcon } from './FingerprintError';
+export { default as FingerprintPadlockIcon } from './FingerprintPadlock';
+export { default as KeyIcon } from './Key';
+export { default as KeyPadlockIcon } from './KeyPadlock';
+export { default as KeyPasswordIcon } from './KeyPassword';
+export { default as LockerIcon } from './Locker';
+export { default as LogInIcon } from './LogIn';
+export { default as LogOutIcon } from './LogOut';
+export { default as PadlockUnlockIcon } from './PadlockUnlock';
+export { default as ShieldErrorIcon } from './ShieldError';
```
Comment (lucianomlima) [inline src/components/Icons/Security/index.ts:16]: 🔧 
```suggestion
export { default as ShieldCheckIcon } from './ShieldCheck';
```

File: src/components/Icons/Security/index.ts [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+export { default as ShieldInfoIcon } from './ShieldInfo';
+export { default as ShieldOffIcon } from './ShieldOff';
+export { default as ShienldErrorIcon } from './ShienldError';
```
Comment (lucianomlima) [inline src/components/Icons/Security/index.ts:19]: 🔧 
```suggestion
export { default as ShieldErrorIcon } from './ShieldError';
```

**Review by lucianomlima — COMMENTED** [review]
I don't know if you do this, but a good practice is review your own PR to catch these little details I comment below 😉

File: src/components/Icons/Security/LogIn.tsx [hunk 6]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+
+import BaseIcon from '../BaseIcon';
+import { getFill, getSize, getViewBox } from '../utils';
+import type { IconProps } from '../types';
+
+function LogIn({
```
Comment (julianosbento) [inline src/components/Icons/Security/LogIn.tsx:9]: 🔧 

```suggestion
function Login({
```

File: src/components/Icons/Security/LogIn.tsx [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+  id = 'LogInIcon',
```
Comment (julianosbento) [inline src/components/Icons/Security/LogIn.tsx:10]: 🔧 

```suggestion
  id = 'LoginIcon',
```

File: src/components/Icons/Security/LogIn.tsx [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+      />
+      <Path
+        d="M16.6023 12.7005C16.6949 12.6062 16.7651 12.498 16.8128 12.3828C16.8613 12.2657 16.8883 12.1375 16.8887 12.003L16.8887 12L16.8887 11.997C16.8879 11.7421 16.7903 11.4874 16.5958 11.2929L12.5958 7.29289C12.2053 6.90237 11.5721 6.90237 11.1816 7.29289C10.791 7.68342 10.791 8.31658 11.1816 8.70711L13.4745 11H3.88867C3.33639 11 2.88867 11.4477 2.88867 12C2.88867 12.5523 3.33639 13 3.88867 13H13.4745L11.1816 15.2929C10.791 15.6834 10.791 16.3166 11.1816 16.7071C11.5721 17.0976 12.2053 17.0976 12.5958 16.7071L16.5951 12.7078L16.6023 12.7005Z"
+        fill={fill}
+      />
+    </BaseIcon>
+  );
+}
+
+export default LogIn;
```
Comment (julianosbento) [inline src/components/Icons/Security/LogIn.tsx:33]: 🔧 

```suggestion
export default Login;
```

File: src/components/Icons/Security/LogOut.tsx [hunk 9]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+
+import BaseIcon from '../BaseIcon';
+import { getFill, getSize, getViewBox } from '../utils';
+import type { IconProps } from '../types';
+
+function LogOut({
```
Comment (julianosbento) [inline src/components/Icons/Security/LogOut.tsx:9]: 🔧 

```suggestion
function Logout({
```

File: src/components/Icons/Security/LogOut.tsx [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+  id = 'LogOutIcon',
```
Comment (julianosbento) [inline src/components/Icons/Security/LogOut.tsx:10]: 🔧 

```suggestion
  id = 'LogoutIcon',
```

File: src/components/Icons/Security/LogOut.tsx [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+      />
+      <Path
+        d="M22.6023 12.7005C22.6949 12.6062 22.7651 12.498 22.8128 12.3828C22.8613 12.2657 22.8883 12.1375 22.8887 12.003L22.8887 12L22.8887 11.997C22.8879 11.7421 22.7903 11.4874 22.5958 11.2929L18.5958 7.29289C18.2053 6.90237 17.5721 6.90237 17.1816 7.29289C16.791 7.68342 16.791 8.31658 17.1816 8.70711L19.4745 11H9.88867C9.33639 11 8.88867 11.4477 8.88867 12C8.88867 12.5523 9.33639 13 9.88867 13H19.4745L17.1816 15.2929C16.791 15.6834 16.791 16.3166 17.1816 16.7071C17.5721 17.0976 18.2053 17.0976 18.5958 16.7071L22.5951 12.7078L22.6023 12.7005Z"
+        fill={fill}
+      />
+    </BaseIcon>
+  );
+}
+
+export default LogOut;
```
Comment (julianosbento) [inline src/components/Icons/Security/LogOut.tsx:35]: 🔧 

```suggestion
export default Logout;
```

File: src/components/Icons/Security/PadlockUnlock.tsx [hunk 12]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+
+import BaseIcon from '../BaseIcon';
+import { getFill, getSize, getViewBox } from '../utils';
+import type { IconProps } from '../types';
+
+function PadlockUnlock({
```
Comment (julianosbento) [inline src/components/Icons/Security/PadlockUnlock.tsx:9]: ⛏️

```suggestion
function PadlockOpen({
```

File: src/components/Icons/Security/PadlockUnlock.tsx [hunk 13]
```diff
@@ -0,0 +1,29 @@
...
+  id = 'PadlockUnlockIcon',
```
Comment (julianosbento) [inline src/components/Icons/Security/PadlockUnlock.tsx:10]: ⛏️

```suggestion
  id = 'PadlockOpenIcon',
```

File: src/components/Icons/Security/PadlockUnlock.tsx [hunk 14]
```diff
@@ -0,0 +1,29 @@
...
+    <BaseIcon id={id} color={color} width={width} height={height} viewBox={viewBox} {...props}>
+      <Path
+        d="M12.8887 4C11.241 4 9.88867 5.35228 9.88867 7V10H18.8887C19.9932 10 20.8887 10.8954 20.8887 12V20C20.8887 21.1046 19.9932 22 18.8887 22H6.88867C5.7841 22 4.88867 21.1046 4.88867 20V12C4.88867 10.8954 5.7841 10 6.88867 10H7.88867V7C7.88867 4.24772 10.1364 2 12.8887 2C15.641 2 17.8887 4.24772 17.8887 7C17.8887 7.55228 17.441 8 16.8887 8C16.3364 8 15.8887 7.55228 15.8887 7C15.8887 5.35228 14.5364 4 12.8887 4ZM6.88867 12V20H18.8887V12H6.88867Z"
+        fill={fill}
+      />
+    </BaseIcon>
+  );
+}
+
+export default PadlockUnlock;
```
Comment (julianosbento) [inline src/components/Icons/Security/PadlockUnlock.tsx:29]: ⛏️

```suggestion
export default PadlockOpen;
```

File: src/components/Icons/Security/Faceid.tsx [hunk 15]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+
+import BaseIcon from '../BaseIcon';
+import { getFill, getSize, getViewBox } from '../utils';
+import type { IconProps } from '../types';
+
+function Faiced({
```
[the remark this answers is not in the document]
Comment (Michele-Camargo) [reply src/components/Icons/Security/Faceid.tsx]: comment resolved in https://github.com/magnetis/astro-native/pull/583/commits/a89a68785dc256b4185c1d475dc11d45b1f65917

File: src/components/Icons/Security/PadlockLock.tsx (hunk 1 above)
Comment (Michele-Camargo) [reply src/components/Icons/Security/PadlockLock.tsx]: comment resolved in https://github.com/magnetis/astro-native/pull/583/commits/a89a68785dc256b4185c1d475dc11d45b1f65917

File: src/components/Icons/Security/ShieldError.tsx (hunk 2 above)
Comment (Michele-Camargo) [reply src/components/Icons/Security/ShieldError.tsx]: comment resolved in https://github.com/magnetis/astro-native/pull/583/commits/a89a68785dc256b4185c1d475dc11d45b1f65917

File: src/components/Icons/Security/ShienldError.tsx (hunk 3 above)
Comment (Michele-Camargo) [reply src/components/Icons/Security/ShienldError.tsx]: comment resolved in https://github.com/magnetis/astro-native/pull/583/commits/a89a68785dc256b4185c1d475dc11d45b1f65917

File: src/components/Icons/Security/index.ts (hunk 4 above)
Comment (Michele-Camargo) [reply src/components/Icons/Security/index.ts]: comment resolved in https://github.com/magnetis/astro-native/pull/583/commits/a89a68785dc256b4185c1d475dc11d45b1f65917

File: src/components/Icons/Security/index.ts (hunk 5 above)
Comment (Michele-Camargo) [reply src/components/Icons/Security/index.ts]: comment resolved in https://github.com/magnetis/astro-native/pull/583/commits/a89a68785dc256b4185c1d475dc11d45b1f65917

File: src/components/Icons/Security/LogIn.tsx (hunk 6 above)
Comment (Michele-Camargo) [reply src/components/Icons/Security/LogIn.tsx]: comment resolved in https://github.com/magnetis/astro-native/pull/583/commits/a89a68785dc256b4185c1d475dc11d45b1f65917

File: src/components/Icons/Security/LogIn.tsx (hunk 7 above)
Comment (Michele-Camargo) [reply src/components/Icons/Security/LogIn.tsx]: comment resolved in https://github.com/magnetis/astro-native/pull/583/commits/a89a68785dc256b4185c1d475dc11d45b1f65917

File: src/components/Icons/Security/LogIn.tsx (hunk 8 above)
Comment (Michele-Camargo) [reply src/components/Icons/Security/LogIn.tsx]: comment resolved in https://github.com/magnetis/astro-native/pull/583/commits/a89a68785dc256b4185c1d475dc11d45b1f65917

File: src/components/Icons/Security/LogOut.tsx (hunk 9 above)
Comment (Michele-Camargo) [reply src/components/Icons/Security/LogOut.tsx]: comment resolved in https://github.com/magnetis/astro-native/pull/583/commits/a89a68785dc256b4185c1d475dc11d45b1f65917

File: src/components/Icons/Security/LogOut.tsx (hunk 10 above)
Comment (Michele-Camargo) [reply src/components/Icons/Security/LogOut.tsx]: comment resolved in https://github.com/magnetis/astro-native/pull/583/commits/a89a68785dc256b4185c1d475dc11d45b1f65917

File: src/components/Icons/Security/LogOut.tsx (hunk 11 above)
Comment (Michele-Camargo) [reply src/components/Icons/Security/LogOut.tsx]: comment resolved in https://github.com/magnetis/astro-native/pull/583/commits/a89a68785dc256b4185c1d475dc11d45b1f65917

File: src/components/Icons/Security/PadlockUnlock.tsx (hunk 12 above)
Comment (Michele-Camargo) [reply src/components/Icons/Security/PadlockUnlock.tsx]: comment resolved in https://github.com/magnetis/astro-native/pull/583/commits/a89a68785dc256b4185c1d475dc11d45b1f65917

File: src/components/Icons/Security/PadlockUnlock.tsx (hunk 13 above)
Comment (Michele-Camargo) [reply src/components/Icons/Security/PadlockUnlock.tsx]: comment resolved in https://github.com/magnetis/astro-native/pull/583/commits/a89a68785dc256b4185c1d475dc11d45b1f65917

File: src/components/Icons/Security/PadlockUnlock.tsx (hunk 14 above)
Comment (Michele-Camargo) [reply src/components/Icons/Security/PadlockUnlock.tsx]: comment resolved in https://github.com/magnetis/astro-native/pull/583/commits/a89a68785dc256b4185c1d475dc11d45b1f65917

**Review by lucianomlima — APPROVED** [review]
You use the wrong filenames in Security index file.

Captura de Tela 2022-05-19 às 19 52 36

Besides that, LGTM

**Review by julianosbento — APPROVED** [review]

**Comment by Michele-Camargo** [comment]
This PR will be closed. An improper `rebase`/`git push` was done and brought in some commits from `astro-native@next`. With that we created another branch and did a `git cherry pic`.

**Pull request closed without merging**